src/com/xilinx/rapidwright/util/BlackboxPopulator.java

```diff
@@ -65,6 +64,12 @@ public class RelocateModulesIntoBlackboxes {
     public static boolean relocateModuleInsts(Design top, Module mod, String cellAnchor, List<Pair<String, String>> blackboxes) {
         System.out.println("\n\nRelocate " + mod.getName());
 
+        // Make the blackbox whose reference INT tile the same as the implementation as the last to be copied.
```

Comment:
This sentence doesn't make any sense to me. Can you reword please?

Reply:
If I understand this comment and the line below, perhaps we have fixed this as of the recent change on Friday.  Can we test to see if we can removing the sorting procedure?

Reply:
Remove this sorting. The dcp outputs show the same report_route_status result with or without sorting.  They are many changes from that point and now. Don't know which changes rendering this unnecessary.